README.md

```diff
@@ -32,10 +32,10 @@ at the Singapore Haskell meetup: http://typeful.net/talks/hpack
 
 ## Examples
 
- * Given this [package.yaml](https://github.com/sol/hpack/blob/master/package.yaml) running `hpack` will generate [hpack.cabal](https://github.com/sol/hpack/blob/master/hpack.cabal)
- * Given this [package.yaml](https://github.com/zalora/getopt-generics/blob/master/package.yaml) running `hpack` will generate [getopt-generics.cabal](https://github.com/zalora/getopt-generics/blob/master/getopt-generics.cabal)
- * Given this [package.yaml](https://github.com/hspec/sensei/blob/master/package.yaml) running `hpack` will generate [sensei.cabal](https://github.com/hspec/sensei/blob/master/sensei.cabal)
- * Given this [package.yaml](https://github.com/haskell-compat/base-orphans/blob/master/package.yaml) running `hpack` will generate [base-orphans.cabal](https://github.com/haskell-compat/base-orphans/blob/master/base-orphans.cabal)
+ * Given this [package.yaml](https://github.com/sol/hpack/blob/main/package.yaml) running `hpack` will generate [hpack.cabal](https://github.com/sol/hpack/blob/main/hpack.cabal)
+ * Given this [package.yaml](https://github.com/zalora/getopt-generics/blob/main/package.yaml) running `hpack` will generate [getopt-generics.cabal](https://github.com/zalora/getopt-generics/blob/main/getopt-generics.cabal)
```

Comment:
This repo still uses `master`, albeit the repo itself mas moved to https://github.com/soenkehahn/getopt-generics/.